tests/unit/challenges/test_models.py

```diff
@@ -115,25 +148,6 @@ class ChallengePhaseTestCase(BaseTestCase):
 
     def setUp(self):
         super(ChallengePhaseTestCase, self).setUp()
-        try:
```

Comment:
@isht3 : I am not able to understand why is this piece of code touched in this me PR. Can you please explain me ?